src/service/get-tag.js

```diff
@@ -1,5 +1,5 @@
-import tags from '@/service/data/tags.json';
+import Tags from '@/service/data/tags.json';
```

Comment:
It might be good to intentionally use all small-caps for variables holding the data object so that it's not the same with React objects. I maybe wrong, but let me know what you think.